src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    setIsLoading(true);
+    setIsError(false);
+    getPeople()
+      .then(setPeople)
+      .catch(() => setIsError(true))
+      .finally(() => setIsLoading(false));
```

Comment:
This is a code quality issue. You are repeating yourself when you set `isLoading` and `isError` to `true` and `false` respectively. You could improve this by creating a `startLoading` and `endLoading` function that sets these states appropriately. This would make your code more readable and less prone to errors.